go/vt/callinfo/plugin_mysql.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The Vitess Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreedto in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package callinfo
```

Comment:
Why a new package instead of adding it to `go/mysql`?

Reply:
It's not a "new" package.

The model is that there's a common `callinfo` abstraction with plugins that implement handling support for the specific transport protocols. The only existing one is grpc (though I assume Google has a stubby one) so this adds mysql as a second implementation of the model.

Reply:
ah I see. That makes sense.